homeassistant/helpers/config_entry_oauth2_flow.py

```diff
@@ -167,6 +167,7 @@ async def async_generate_authorize_url(self, flow_id: str) -> str:
             URL(self.authorize_url)
             .with_query(
                 {
+                    "appSelect": 1,
```

Comment:
Integrations should implement `extra_authorize_data` if more query parameters to the authorize url are needed.

Reply:
Thanks for letting me know @MartinHjelmare. This has been updated.